templates/Generalized_Masking.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
<p>Piggybacking off of a comment from <code>ark.utils.masking_utils.py</code>, I'd prefer leaving <code>cluster_col</code> as a hidden default argument in <code>generate_cell_masks</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Similar to the spatial scripts, this cell masking would often be used for a more broad cell type grouping specified by the user previously, so rarely would we use the 'cell_meta_cluster' column. It's just set here as a example!</p>